skimage/feature/setup.py

```diff
@@ -11,7 +11,8 @@ def configuration(parent_package='', top_path=None):
 
     config = Configuration('feature', parent_package, top_path)
     config.add_data_dir('tests')
-
+    config.add_data_files('orb_descriptor_positions.txt')
```

Comment:
orb file was moved to the directory where it belongs. These are more like coefficient positions anyway as opposed to data.